common/changes/office-ui-fabric-react/jessecar-SvgSupport_2017-10-17-16-15.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "Add a mechanism to provide a Fabric-wide icon renderer function (as opposed to the control-specific onRenderIcon fn",
```

Comment:
The convention is to make the comment [ComponentName]: feature, so can you just prefix the message with Icon: ?